src/FsUnit.Xunit/CustomMatchers.fs

```diff
@@ -8,6 +8,16 @@ open System.Reflection
 
 let equal x = CustomMatcher<obj>(sprintf "Equals %A" x, fun a -> a = x)
 
+let equivalent f x = CustomMatcher<obj>(sprintf "Equivalent to %A" x, fun c -> 
+                                                                try 
+                                                                    let toCollection o = o |> Seq.cast |> Seq.toArray :> ICollection
+                                                                    match c with
+                                                                        | :? ICollection as i -> f (toCollection x) i
+                                                                        | :? System.Collections.IEnumerable as e -> f (toCollection x) (toCollection e)
+                                                                    true
+                                                                with
+                                                                | ex -> raise ex)
+
```

Comment:
Why do no you have tests for this?

Reply:
There're none? I am testing with sequences, arrays and lists. Should I test with Enumerable, too?
I see there are no tests for the failing part.

Is this the right way to implement this or do I have to match with `seq<_>`, `list<_>` and `array<_>`, too?

Reply:
But this `let equivalent f x = CustomMatcher<obj>` is implementation for xUnit, is not it?

Reply:
This `CustomMatchers.fs` is referenced in `FsUnit.Xunit` and in `FsUnit.MsTestUnit` and is located in the folder of `FsUnit.Xunit`